inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -20,6 +20,9 @@ std::vector<std::string> disabledTestPatterns() {
         // TODO: Issue 32023
         R"(.*(QuantGroupConvBackpropData2D)*QG=Pertensor.*)",
         // TODO: Issue 31845
-        R"(.*(FakeQuantize).*)"
+        R"(.*(FakeQuantize).*)",
+        // TODO: Issue: 32521
+        R"(.*(EltwiseLayerTest).*secondaryInputType=CONSTANT.*netPRC=FP16.*)",
+        R"(.*(EltwiseLayerTest).*IS=.*1.1.1.1.*opType=SCALAR.*)"
```

Comment:
JIRA ticket number

Reply:
check if these are valid fails when eltwise.cpp is fixed, also amke sure that these are precise enough if I recall it is failing cause broadcast blobs are limited, so there is chance that it will pass on Prod or/and Parameter as secondary input